### PR TITLE
Symfony 5 support

### PR DESCRIPTION
**Pull request by napestershine**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/framework-bundle": "^2.8|^3.0|^4.0",
+        "symfony/framework-bundle": "^2.8|^3.0|^4.0|^5.1",
         "symfony/doctrine-bridge": "^2.8|^3.0|^4.0",
         "symfony/dependency-injection": "^2.8|^3.0|^4.0",
```
Comment (weaverryan) [inline composer.json:18]: We need to update to `^5.0` on all the `symfony/` libraries, including in require-dev (and I would use `^5.0`... I think it's not a big deal to still support that version).

Also, it's time to drop everything below 4.4. So I would use `"^4.4|^5.0"`

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
```
Comment (weaverryan) [inline composer.json:15]: And probably time to bump this to `"php": ">=7.1.3",` which is the minimum for Symfony 4.4

File: composer.json (hunk 1 above)
Comment (napestershine) [reply composer.json:18]: Hi, yes sure. I will update this PR today. I just started it only.

Comment (napestershine) [reply composer.json]: Updated. Thanks for review.

File: composer.json (hunk 2 above)
Comment (napestershine) [reply composer.json]: Updated. Thanks for review.

**Comment by kbond** [comment]
Thanks @napestershine, don't worry about the failing travis build - it's a known issue I need to look into.

@weaverryan should we keep Symfony 3.4 support?

**Comment by kbond** [comment]
Never mind, I see Ryan suggested only supporting 4.4+. I'm good with that.

@napestershine, can you change the symfony package constraints from `^4.0|^5.0` to `^4.4|^5.0`?

**Comment by napestershine** [comment]
@kbond I have updated it.

**Comment by kbond** [comment]
Thanks, tagged [v3.4.0](https://github.com/kbond/ZenstruckRedirectBundle/releases/tag/v3.4.0).

**Comment by napestershine** [comment]
@kbond I am getting exact same errors on my installation shown by Travis.

**Comment by kbond** [comment]
I see a lot of different travis errors - can you link to the specific one?

**Comment by kbond** [comment]
@napestershine, I refactored and got the CI tests working again - can you try [v3.5.0](https://github.com/kbond/ZenstruckRedirectBundle/releases/tag/v3.5.0)?

**Comment by napestershine** [comment]
Yes sure. I will try it and update you. Thanks

**Pull request merged**